RedHatInsights/insights-inventory-frontend
# chore(tests): Update coverage config

This should fix the coverage reporting to codcov.

**How to test:**

* Run `rm -rf node_modules && npm install && npm run travis:verify`
* The HTML reports in coverage, coverage-cypress, and coverage-jest to make sense.
